main.py

```diff
@@ -55,16 +73,25 @@ def main(k, color_tol, size):
                             k=k, color_tol=color_tol)
                     except NoArtworkException:
                         r, g, b = 255, 255, 255
-                    led.set_color(r, g, b)
+                    if WS281X['is_active'] == "True":
```

Comment:
Minor detail but the convention in this project is that single quotes are used for strings.